unfoldingWord/translationCore
# Feature/add tc version to manifest

#### This pull request addresses:

Issue: https://github.com/unfoldingWord-dev/translationCore/issues/3322
Also fixes ts version 3 manifest migration: #3366


#### How to test this pull request:

Either do local import of tstudio project or online import of bible project.  After import manifest.json file should have tc_version entry of 1. 


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3367)
